src/helpers.js

```diff
@@ -0,0 +1,32 @@
+var helpers = {
+    arrayToObject: function(array, keyField) {
+        var newObj = array.reduce(function(obj, item) {
+            obj[item[keyField]] = item;
+            return obj;
+        }, {});
+    return newObj;
+    },
+    loadScript: function(src, callback) {
+        var script = document.createElement('script');
+        script.type = 'text/javascript';
+      
+        // IE
```

Comment:
when you put IE here, do you mean support for IE 8 and before? on mdn i'm seeing [this](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload), which shows .onload to work on IE9 and current. if you are trying to support IE8 and before, then we can kill this. our [core sdk supports ie10+](https://docs.mparticle.com/developers/sdk/web/browser-compatibility/) and up right now.

Reply:
got it - will remove!